src/Raven.Server/Documents/DocumentDatabase.cs

```diff
@@ -329,6 +328,11 @@ record = _serverStore.Cluster.ReadDatabase(context, Name, out index);
                 if (record == null)
                     DatabaseDoesNotExistException.Throw(Name);
 
+                if (record!.SupportedFeatures == null || record.SupportedFeatures.Count == 0)
+                    SupportedFeatures = new List<string> { "ThrowRevisionKeyTooBigFix" };
```

Comment:
why not always simply take what we have from the database record?

Reply:
because for new databases we would like to set `SupportedFeatures` here

Reply:
but this should be handled in `CreateDatabase`, no?

Reply:
done